app/components/Swimlane/style.scss

```diff
@@ -32,6 +32,41 @@ $swimlane-height-small: 250px;
     margin-left: 0;
   }
 
+  &--collapsed {
+    border-bottom-left-radius: $swimlane-border-radius;
+    border-bottom-right-radius: $swimlane-border-radius;
+    flex: none;
+    height: $swimlane-width;
+    overflow-y: visible;
+    width: $swimlane-header-height;
+
+    .swimlane {
+      &__cards {display: none;}
+
+      &__header {
+        border-bottom-left-radius: $swimlane-border-radius;
+        border-bottom-right-radius: $swimlane-border-radius;
+        height: $swimlane-width;
+        width: $swimlane-header-height;
+
+        form {
```

Comment:
For the most part we try to stay away from using html elements in our css. Instead we try to use [BEM](http://getbem.com/).